DEVELOPMENT.md

```diff
@@ -1,40 +1,80 @@
 # Development
 
-## Testing via examples
-
-1. Add a new example project in `examples` folder. It is a regular NPM package with its own `package.json` and Cypress dev dependency.
-1. Add a corresponding `.github/workflows` YAML file that uses this action and runs inside `examples/X` working directory. The example should demonstrate the feature
-1. Add workflow badge to README if needed
-
-## Testing against another repo
-
-1. Create a new local branch for any development, for example `git checkout -b featureA`
-1. Update the source code in [index.js](index.js)
-1. Build `dist` file(s) using `npm run build`
-1. Commit any changed files, note the SHA of the commit
-1. Push the local branch to GitHub
-1. In a test repository, create a test branch, change the action to point at the new commit. Example
-
-```yml
-name: End-to-end tests
-on: push
-jobs:
-  cypress-run:
-    runs-on: 22.04
-    steps:
-      - uses: actions/checkout@v3
-      # use specific commit pushed to GitHub
-      - uses: cypress-io/github-action@<full commit SHA>
+This document describes topics useful to contributors to this repository. The repository's purpose is to provide a GitHub JavaScript action which is published to:
+
+1. npm's JavaScript Package Registry as [@cypress/github-action](https://www.npmjs.com/package/@cypress/github-action)
+2. GitHub's Marketplace as [cypress-io/github-action](https://github.com/marketplace/actions/cypress-io#cypress-iogithub-action--)
+
+You can read the GitHub document [Creating a JavaScript action](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) for background information on how JavaScript actions for GitHub are created and how they work.
+
+## Providing fixes or features
+
+If you are submitting a Pull Request (PR) to provide a fix or feature for the action, the following is relevant to you:
+
+The main source code elements for the action are:
+- [action.yml](action.yml)
+- [index.js](index.js)
+- [src/ping.js](src/ping.js)
+
+The action runs from the [dist](dist) directory, which requires a `build` step to set up. This step uses [@vercel/ncc](https://github.com/vercel/ncc) to compile the code and modules into one file used for distribution. See below:
+
+1. If you are a new external contributor, then first fork the repository (see GitHub documentation [About forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)).
+1. Create a new branch with a meaningful name e.g. `fix/my-bug` based on the current `master` branch.
+1. Make the necessary source code changes, including additions or changes to the [README.md](./README.md) documentation if parameters are added or affected.
+1. Ensure you are using [Node.js LTS](https://nodejs.org/en).
+1. Execute the following in the root directory of the cloned repository
```

Comment:
```suggestion
1. Execute the following in the root directory of the cloned repository after change has been made:
```

Reply:
@emilyrohrbough 

Perhaps I need to be more specific in the introduction to the list that the items are intended to be carried out in the order that they are listed? The intention was for step 3 (make changes) to be carried out before step 5 (run format and build):

![image](https://user-images.githubusercontent.com/66998419/230075282-92fa5890-dd4c-4750-8356-ac192f5b0798.png)